lib/Controller/ConfigController.php

```diff
@@ -23,6 +24,7 @@ public function __construct(
 		string   $appName,
 		IRequest $request,
 		private IConfig  $config,
+		UtilsService $utilsService,
```

Comment:
```suggestion
		private UtilsService $utilsService,
```
to automatically make it a class property